cli/cli/Services/BeamoLocalSystem.cs

```diff
@@ -108,22 +107,29 @@ public BeamoLocalSystem(ConfigService configService, IAppContext ctx, IRealmsApi
 	/// Get list of <see cref="BeamoId"/>s that this service depends on.
 	/// </summary>
 	/// <param name="beamoServiceId">The identifier of the Beamo service.</param>
-	/// <param name="projectExtension">The extension of the project files. Default is "csproj".</param>
 	/// <returns>Returns a list of <see cref="BeamoId"/>s that this service depends on.</returns>
-	public async Task<List<string>> GetDependencies(string beamoServiceId, string projectExtension = "csproj")
+	public async Task<List<string>> GetDependencies(string beamoServiceId)
 	{
 		var serviceDefinition = BeamoManifest.ServiceDefinitions.FirstOrDefault(s => s.BeamoId == beamoServiceId);
 		if (string.IsNullOrWhiteSpace(serviceDefinition?.ProjectDirectory))
 		{
 			return new List<string>();
 		}
+
+		var projectExtension = serviceDefinition.Language switch

```

Comment:
nitpick- I'd move this onto the `serviceDefinition` as a computed property

Reply:
Will do 👍